src/guides/flash-cm4-firmware.html

```diff
@@ -18,9 +18,17 @@
   <ol start="1">
     <li>
       <span>Open the procedure <a
-        href="https://yellow.home-assistant.io/guides/reinstall-os/"
+        href="/guides/reinstall-os/"
         >How to re-install the operating system.</a
-      ><li>Follow steps 1&ndash;6 of section <b>Reinstall Home Assistant OS using rpiboot</b>.</li></span>
+      ><li>If you have a Home Assistant Yellow version 1.3 or later: Follow steps 1&ndash;7 of section <a
+        href="/guides/reinstall-os/#install-os-rpiboot"
+        >Reinstall Home Assistant OS using rpiboot</a
+      >.</li>
+      <li>If you have a Home Assistant Yellow version 1.2 or older: Follow steps 1&ndash;6 of section <a
+        href="/guides/reinstall-os/#install-os-rpiboot"
+        >Reinstall Home Assistant OS using rpiboot</a
+      >.</li>
+    </span>
```

Comment:
The jumper only needs to be set right after power up, this is also true when flashing the CM4 EEPROM. Hence, you can just use steps 1-7 for both variants here, and drop the unbridge step below.